docs/app-dev/abci-spec.md

```diff
@@ -111,14 +111,16 @@ See below for more details on the message types and how they are used.
   - `Time (google.protobuf.Timestamp)`: Genesis time.
   - `ChainID (string)`: ID of the blockchain.
   - `ConsensusParams (ConsensusParams)`: Initial consensus-critical parameters.
-  - `Validators ([]Validator)`: Initial genesis validators.
+  - `Validators ([]ValidatorUpdate)`: Initial genesis validators.
   - `AppStateBytes ([]byte)`: Serialized initial application state. Amino-encoded JSON bytes.
 - **Response**:
   - `ConsensusParams (ConsensusParams)`: Initial
     consensus-critical parameters.
-  - `Validators ([]Validator)`: Initial validator set.
+  - `Validators ([]ValidatorUpdate)`: Initial validator set.
 - **Usage**:
   - Called once upon genesis.
+  - List of validators in the response will be used as the initial validator set
```

Comment:
what if it (validators list in the response) is empty? or nil?

Reply:
Good catch will update